docs/api/zh/geometries/ShapeGeometry.html

```diff
@@ -15,7 +15,7 @@ <h1>形状缓冲几何体（[name]）</h1>
 
 		<iframe id="scene" src="scenes/geometry-browser.html#ShapeGeometry"></iframe>
 
-		<script>
+				<script>
```

Comment:
Please undo this change.

In general, please do not touch translated pages.

Reply:
i have changed it that was a mistake.